fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml

```diff
@@ -366,7 +366,7 @@ paths:
   /v1/pis/banks/{bank-id}/accounts/{account-id}/payments/single:
     get:
       summary: Ask for all payments of this account
-      description: This method is used to initiate a payment at the Fintech Server.
+      description: This method is used to get payment status.
```

Comment:
Not for this place, but we need to include `X-Pis-Psu-Authentication-Required` in the signing procedure, just like it is done for payments. See:
`PaymentInitiationDataToSign` and usages of `psuAuthenticationRequired`, also `HttpRequestToDataToSignMapper` `mapToPaymentInititation` for how-to

Reply:
removed